models/note.go

```diff
@@ -16,7 +16,7 @@ const (
 	PREDICTIONS
 )
 
-const noteTypeStrings = [...]string{
+var noteTypeStrings = [...]string{
```

Comment:
i get a compilation when its const: `models/note.go:19:7: const initializer [4]string literal is not a constant`. Google tells me its cause you can't make arrays const, since they are mutable by nature.